app/src/main/java/edu/uc/forbesne/shoppingsidekick/ui/main/MainFragment.kt

```diff
@@ -27,4 +23,7 @@ class MainFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
```

Comment:
Moved companion object to bottom of file. Example: https://kotlinlang.org/docs/coding-conventions.html#overload-layout